cmd/pdcli/alerts/cmd.go

```diff
@@ -105,15 +148,25 @@ func alertsHandler(cmd *cobra.Command, args []string) error {
 		alerts = append(alerts, incidentAlerts...)
 	}
 
-	printAlerts(alerts)
+	// Check for interactive mode
+	if options.interactive {
+
+		err = selectIncident(client)
+
+		if err != nil {
+			return err
+		}
+
+	} else {
+		printAlerts(alerts)
+	}
 
 	return nil
 }
 
-// getIncidents returns a string slice consisting IDs of the first 10 incidents.
-func GetIncidents(c client.PagerDutyClient) ([]string, error) {
+// GetIncidents returns an array pagerduty incidents.
+func GetIncidents(c client.PagerDutyClient) ([]pdApi.Incident, error) {
```

Comment:
I would keep this in a separate package(pdcli/incidents)  for reusability purpose, assuming the signature would be something like 
```func GetIncidents(c client.PagerDutyClient, options pdApi.ListIncidentsOptions) ([] pdApi.Incident, error)```

Reply:
General feedback: Try to lighten up the weight for cmd.go, try to see which things you can decouple and organize

Reply:
Incidents and alerts are inter-chained, so wouldn't it be better to let `GetIncidents` be in the same file as alerts?

Reply:
Suppose I move it into a separate package, incidents are fetched based on the `pdcli alerts` options, replicating options struct in the other pkg, wouldn't that become more complex?

Reply:
Adding to that, to lighten up the weight, shall I move the select functions to a separate pdcli/prompt pkg? And the `clusterLogin` to pdlci/ocm?